tests/_plugins/ui/_impl/test_table.py

```diff
@@ -253,7 +257,7 @@ def test_value_with_sorting_then_selection() -> None:
             page_number=0,
         )
     )
-    assert list(table._convert_value(["0"])) == [
+    assert list(table._convert_value([{"rowId": "0"}])) == [
```

Comment:
i don't think we should add more memory pressure if we don't need. a list of numbers or strings is fine for row selection